modules/server_new/src/main/scala/observe/server/keywords/ConditionOps.scala

```diff
@@ -3,54 +3,153 @@
 
 package observe.server.keywords
 
-import lucuma.core.enums.{Site, SkyBackground, WaterVapor}
+import cats.data.NonEmptyList
+import cats.syntax.all.*
+import lucuma.core.enums.CloudExtinction
+import lucuma.core.enums.ImageQuality
+import lucuma.core.enums.Site
+import lucuma.core.enums.SkyBackground
+import lucuma.core.enums.WaterVapor
+import lucuma.core.math.Angle
+import lucuma.core.math.Wavelength
 import lucuma.core.model.ConstraintSet
 import observe.model.Conditions
 
+import scala.annotation.tailrec
+import scala.math.Ordering.Implicits.infixOrderingOps
+import scala.math.Pi
+import scala.math.abs
+import scala.math.pow
+import scala.math.sin
+
 object ConditionOps {
-  
+
   extension (wv: WaterVapor) {
     def toMillimeters(site: Site): Double = site match
-      case Site.GN => wv match
-        case WaterVapor.VeryDry => 1.0
-        case WaterVapor.Dry => 1.6
-        case WaterVapor.Median => 3.0
-        case WaterVapor.Wet => 5.0
-      case Site.GS => wv match
-        case WaterVapor.VeryDry => 2.3
-        case WaterVapor.Dry => 4.3
-        case WaterVapor.Median => 7.6
-        case WaterVapor.Wet => 10.0
+      case Site.GN =>
```

Comment:
Do you think we should put things like this on db? Such that we can change them without rebuilding?

Reply:
I don't know how frequently those values change. It is a question fro Andy.